src/index.html

```diff
@@ -7,12 +7,70 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.
+      com/css2?family=Poppins:ital,wght@0,100;
+      0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;
+      1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;
+      1,900&family=Roboto:ital,wght@0,100;0,300;0,400;
+      0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;
+      1,900&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="card">
+    <main class="product-card" data-qa="card">
+      <div
+        class="product-card__image">
+      </div>
+
+      <div class="product-card__content">
+        <div class="content__text">
+          <p class="main__text">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="text__code">
+            Product code: 195434
+          </p>
+        </div>
+
+        <div class="content__mark">
+          <div class="content__mark__stars">
+            <ul class="stars stars--N">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+          </div>
+
+          <div class="mark__reviews">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="content__price">
+          <div class="price__text">
+            Price:
+          </div>
+          <div class="price__value">
+            $2,199
+          </div>
+        </div>
+
+        <div class="content__buy">
```

Comment:
The class name `content__buy` should follow the BEM naming convention. Consider using `product-card__content__buy`.